### PR TITLE
Danish translation for installer - Improvement

### PR DESCRIPTION
**Pull request by Casper-O**
| Questions | Answers |
| --- | --- |
| Branch? | Develop |
| Description? | Adding Danish translation for installer + fixing a few typo |
| Type? | improvement |
| Category? | IN |
| BC breaks? | No |
| Deprecations? | No |
| Fixed ticket? |  |
| How to test? | Ask someone that knows danish |

#### Important guidelines
- Make sure [your local branch is up to date](https://help.github.com/articles/syncing-a-fork/) before commiting your changes!
- Your code MUST respect [our Coding Standards](http://doc.prestashop.com/display/PS16/Coding+Standards) (for code written in PHP, JavaScript, HTML/CSS/Smarty/Twig, SQL)!
- Your commit name MUST respect our [naming convention](http://doc.prestashop.com/display/PS16/How+to+write+a+commit+message)!

### CODE REVIEW

**Review by aleeks — APPROVED** [review]

**Comment by AlexEven** [comment]
Hello @Casper-O, 

Thank you for this!!
We've done as you suggested and we're waiting for the feedback of someone who speaks Danish :)
Shouldn't long before we can merge your PR!

Kind regards,

**Comment by Casper-O** [comment]
No Problem. I asked a Danish Prestashop Community to help with the translations. I don't know ho many of them have experience with Github, but hopefully more can help with translation or at least confirming what have been translated

**Review by mowol — CHANGES_REQUESTED** [review]
Customer må skulle oversættes til ental "Kunde".

File: install-dev/langs/da/data/group.xml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0"?>
 <entity_group>
-  <group id="Visitor" name="Visitor"/>
-  <group id="Guest" name="Guest"/>
-  <group id="Customer" name="Customer"/>
+  <group id="Visitor" name="Besøgende"/>
+  <group id="Guest" name="Gæst"/>
+  <group id="Customer" name="Kunder"/>
```
Comment (mowol) [inline install-dev/langs/da/data/group.xml]: bør være Kunde

File: install-dev/langs/da/data/meta.xml [hunk 2]
```diff
@@ -14,9 +14,9 @@
   </meta>
   <meta id="contact" id_shop="1">
     <title>Kontakt os</title>
-    <description>Use our form to contact us</description>
+    <description>Brug din formular til at kontakte os</description>
```
Comment (mowol) [inline install-dev/langs/da/data/meta.xml]: Brug vores formular til at kontakte os

**Review by mowol — CHANGES_REQUESTED** [review]
Sorry for the double post. I have two small change requests to the translation. Otherwise it looks good. Good job Casper!

File: install-dev/langs/da/data/contact.xml [hunk 3]
```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0"?>
 <entity_contact>
   <contact id="Webmaster" name="Webmaster">
-    <description>If a technical problem occurs on this website</description>
+    <description>Hvis et teknisk problem opstår på denne hjemmeside</description>
   </contact>
-  <contact id="Customer_service" name="Customer service">
-    <description>For any question about a product, an order</description>
+  <contact id="Customer_service" name="Kunde service">
```
Comment (tobleronealfen) [inline install-dev/langs/da/data/contact.xml]: Kundeservice
Et ord

File: install-dev/langs/da/data/contact.xml [hunk 4]
```diff
@@ -1,9 +1,9 @@
...
+    <description>For alle spørgsmål om et produkt eller en ordre</description>
```
Comment (tobleronealfen) [inline install-dev/langs/da/data/contact.xml]: Kan gøres mere simpel/meneskeligt med f.eks.
"Spørgsmål vedrørende en vare eller en ordre."

Comment (tobleronealfen) [inline install-dev/langs/da/data/contact.xml]: eller "et produkt"

**Review by Casper-O — COMMENTED** [review]
with help from @tobleronealfen

**Comment by maximebiloe** [comment]
Thank you for this PR!

For your future contributions could look at our [guidelines](http://doc.prestashop.com/display/PS16/How+to+write+a+commit+message) for your commit messages, please?

For this one, I'll squash your commits.

Regards

**Comment by Casper-O** [comment]
Hi @maximebiloe what part of my commit message was not following the guideline? I have been trying to follow the guidelines for all my Commits, but often someone change them, and don't mention why they did it.

My own guess its my naming was not following the guideline this time. But as I said, it often happens that other change it and removing the Category shortcode from it, thats why i did not add it this time

**Comment by maximebiloe** [comment]
For example you commit message was "Update contact.xml" whereas it should be something link "IN: Danish translation for contact fixtures"

**Pull request merged**